smoketest.py

```diff
@@ -0,0 +1,22 @@
+import environ
+import pytest
+from dotenv import load_dotenv
+
+import caselawclient.Client as Client
+from caselawclient.errors import DocumentNotFoundError
+
+load_dotenv()
+env = environ.Env()
+
+api_client = Client.MarklogicApiClient(
+    host=env("MARKLOGIC_HOST"),
+    username=env("MARKLOGIC_USER"),
+    password=env("MARKLOGIC_PASSWORD"),
+    use_https=env("MARKLOGIC_USE_HTTPS", default=None),
+)
+
+
+def test_get_document_type_returns_404():
+    with pytest.raises(DocumentNotFoundError) as e:
+        api_client.get_document_type_from_uri("/not/a/real/url")
```

Comment:
how does this test actually evaluate the the xquery when it is meant to be run in marklogic? Have we set up a marklogic instance to point it to? Are the env vars pointing to staging?

Reply:
This is literally pointing at staging. It's not great, but it's at least something.